conf/broker.conf

```diff
@@ -1258,10 +1258,12 @@ replicatorPrefix=pulsar.repl
 # due to missing ZooKeeper watch (disable with value 0)
 replicationPolicyCheckDurationSeconds=600
 
-# Default message retention time
+# Default message retention time, less than 0 means never expire.
+# Default is 0, which means the data will be removed after all the subscriptions are consumed.
```

Comment:
```suggestion
# Default message retention time.
# The default value is 0, which means the data is removed after all the subscriptions are consumed.
# Value less than 0 means messages never expire.
```

Reply:
Same comments for the other occurrences in this PR.

Reply:
Good!